.travis.yml

```diff
@@ -9,6 +9,10 @@ services:
 install: true
 script:
   - make container
+  - make generate
```

Comment:
let's do a `make check` that just calls hack/check.sh and the script do 
```bash
/bin/env bash

set -xe

make container generate
if [[ -n "$(git status --porcelain)" ]] ;
    then echo "It seems like you need to run `make generate`. Please run it and commit the changes"; git status --porcelain; false;
i
```
This way we can move away from Travis easily

Reply:
ack